storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -693,6 +693,146 @@ func sortTimeSeries(series []prompb.TimeSeries) []prompb.TimeSeries {
 	return series
 }
 
+func TestTranslatorMetricFromOtelMetric(t *testing.T) {
```

Comment:
shouldn't one of these tests result in a metric with _total in the suffix?

Reply:
if it's too much for this fix we can create an issue, but that seems like an important area of coverage to prevent regressions anywhere along the code path from otel to prom metrics